### PR TITLE
Update pipeline playbooks for 3.5 RC2

### PR DESCRIPTION
**Pull request by akofink**

### CODE REVIEW

File: pipelines/pipeline_katello_35.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     forklift_boxes:
       pipeline-katello-3.5-centos7:
         box: centos7
-        memory: 4680
+        memory: 6000
```
Comment (ekohl) [inline pipelines/pipeline_katello_35.yml]: Should we use 6144 so it's a nice 6G?

Comment (akofink) [reply pipelines/pipeline_katello_35.yml]: I don't see why it matters, but I'll go ahead and change it.

**Comment by akofink** [comment]
Updated @ekohl

**Comment by akofink** [comment]
Wait to merge.

**Pull request merged**